### ISSUE TITLE
Support Carthage

### ISSUE THREAD

**Issue by kenstir** [kmcgill88/McPicker-iOS#40]
Thank you for McPicker!  McPicker looks to be exactly what I need, except that it doesn't support Carthage:

```
$ carthage update --platform ios 
*** Skipped building McPicker-iOS due to the error:
Dependency "McPicker-iOS" has no shared framework schemes for any of the platforms: iOS
```

I will experiment with adding support in my forked repo, and if successful, I will file a MR.

**Comment by kmcgill88** [comment]
Did you get this to work with Carthage?

**Comment by kenstir** [comment]
Sadly I did not get it to work.  Sorry for the lack of update.  After a short while of trying, I learned that even if I did get it to work with Carthage, it would not meet my short term need to use Interface Builder to connect a McTextField to my storyboard.  (My understanding is that in order to be @IBDesignable the class cannot be imported via framework.)  So I abandoned the Carthage attempt and included McPicker directly in my source tree.
